board/safety/safety_toyota.h

```diff
@@ -68,6 +84,21 @@ static uint32_t toyota_get_checksum(CANPacket_t *to_push) {
   return (uint8_t)(GET_BYTE(to_push, checksum_byte));
 }
 
+static bool toyota_get_quality_flag_valid(CANPacket_t *to_push) {
+  int addr = GET_ADDR(to_push);
+
+  bool valid = false;
+  if (toyota_lta) {
+    if (addr == 0x260) {
+      // This is always 1 on non-TSS2 platforms
+      valid = GET_BIT(to_push, 3U) == 0U;  // STEER_ANGLE_INITIALIZING
```

Comment:
what's this for?

Reply:
This is a quality flag for the current angle measurement, which we only use to check that the desired angle matches current angle while inactive.